_test/issue-1330.go

```diff
@@ -0,0 +1,43 @@
+package main
+
+import (
+	"fmt"
+	"io"
+	"net"
+)
+
+type wrappedConn struct {
+	net.Conn
+}
+
+func main() {
+	listener, err := net.Listen("tcp", "127.0.0.1:49153")
+	if err != nil {
+		panic(err)
+	}
+	go func() {
+		_, err := listener.Accept()
+		if err != nil {
+			panic(err)
+		}
+	}()
```

Comment:
This part is actually not necessary for our use case. From the client point of view, Dial will succeed as long as there is a Listen on the server side.

Reply:
ok, removed